client/src/components/profile.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import {
+    Form,
+    Button,
+    Badge
+} from 'react-bootstrap';
+import axios from "axios";
+import "../App.css";
+import profilepic from "./maxresdefault.jpg"
+
+class RenderProfilePage extends Component{
+    
+    /*
+    constructor(props){
+        super(props);
+        //this will be changed later on
+    
+        this.state = {
+            userName = "",
+            realName = "",
+            bio = ""
+        };
+        
+    }
+        onUpload(e){
+        e.preventDefault();
+
+        const user = {
+            userName: this.state.userName,
+            realName: this.state.realName
+            bio: this.state.bio
+        };
+        axios
+            .post("http://localhost:8000/api/login", user)
+            .then(res => {
+            console.log(res.data);
+            })
+            .catch(error => {
+            console.log(error);
+            });
+
+        this.setState({ userName: "", realName: "" , bio = ""});
+    }
+*/
+
+    render() {
+        return (
+            <div className="App">
+                <div class="App-header">
+                <h1> MEMESTAGRAM</h1>
+                    <img class = "rcorners1" src={require('./maxresdefault.jpg')} alt = "Matthew McConaughey" height="200" width="240"/>
```

Comment:
Instead of using Matthew McConaughey, would it be possible to use a generic name?

Reply:
the name is for testing purposes but ill make it more generic for my next pull request